### PR TITLE
Signing rate tool

### LINKED ISSUE #270: Create a tool to validate the PoW / Gas configs
It has been noticed that with some configurations its possible to create enough PoW pre-computes to fill blocks. 

We currently have (in testnet) the following network parameters set for the PoW

```
"key": "spam.pow.difficulty",
"value": "15"

"key": "spam.pow.increaseDifficulty",
"value": "0"

"key": "spam.pow.numberOfTxPerBlock"
"value": "100"

"key": "spam.pow.numberOfPastBlocks"
"value": "100"

"key": "network.transactions.maxgasperblock",
"value": "3000000"
```

We need to create a vegatool that is able to get data on how many precomputed PoWs, and for how long the can be used for, for given values for the above network parameters.

The outcome desired is that we can review this data with the core and research teams and define a good base config for both testnet and mainnet.

### PR DESCRIPTION
**Pull request by peterbarrow**
Create a tool that will benchmark the local machine to see how fast it can sign transactions for given difficulty levels.

Closes #270

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,3 +96,6 @@ This creates a market and a set of users and then generates a consistent flow of
 
 ### EventRate
 This listens to an unfiltered event bus stream and reports the number of events arriving per time bucket (default 1 second) and the amount of network bandwidth it used to receive them. The bucket length and the number of historic buckets it uses to generate the average values can be set on the commandline. 
+
+### SigningRate
+This runs a benchmark using the same encryption used to sign each transaction. It reports back the number of transactions per second the machine can sign and can be used to help set the proof of work difficulty value for a network. It will also allow uses to get a feel for the rate in which a wallet service will be able to sign and forward on transactions if run on the same machine.
```
Comment (ze97286) [inline README.md]: it's not really signing nor encryption, it's just generating proof of work

File: cmd/signingrate.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		Use:   "signingrate",
+		Short: "Display the rate at which the current machine can sign transactions",
+		RunE:  runSigningRate,
+	}
+)
+
+func init() {
+	rootCmd.AddCommand(signingRateCmd)
+	signingRateCmd.Flags().IntVarP(&signingRateOpts.MinPoWLevel, "minimumpowdifficulty", "a", 1, "lowest PoW difficulty to test")
+	signingRateCmd.Flags().IntVarP(&signingRateOpts.MaxPoWLevel, "maximumpowdifficulty", "b", 20, "highest PoW difficulty to test")
```
Comment (ze97286) [inline cmd/signingrate.go]: maybe worth going a bit above 20 if it's not too slow to get a real feel

File: README.md (hunk 1 above)
Comment (peterbarrow) [reply README.md]: Urgh, then I've called the tool the wrong name then. Thanks for the feedback Zohar.

Comment (ze97286) [reply README.md]: sorry...

**Review by ze97286 — APPROVED** [review]

**Pull request merged**